sencha-workspace/packages/slate-cbl/src/view/skill/OverviewBody.js

```diff
@@ -0,0 +1,92 @@
+                        '<tpl for=".">',
+                            '<tr class="skill-grid-demo-row" data-demonstration="{ID}">',
+                                '<td class="skill-grid-demo-data skill-grid-demo-index">{[xindex]}</td>',
+                                '<td class="skill-grid-demo-data skill-grid-demo-date">{Demonstration.Demonstrated:date("M j, Y")}</td>',
+                                '<td class="skill-grid-demo-data skill-grid-demo-level"><div class="level-color cbl-level-{Level}"><tpl if="Level==0">M<tpl else>{Level}</tpl></div></td>',
+                                '<td class="skill-grid-demo-data skill-grid-demo-type">{Demonstration.ExperienceType:htmlEncode}</td>',
+                                '<td class="skill-grid-demo-data skill-grid-demo-context">{Demonstration.Context:htmlEncode}</td>',
+                                '<td class="skill-grid-demo-data skill-grid-demo-task">{Demonstration.PerformanceType:htmlEncode}</td>',
+                                // '<td class="skill-grid-demo-data skill-grid-demo-artifact"><tpl if="Demonstration.ArtifactURL"><a href="{Demonstration.ArtifactURL:htmlEncode}">{Demonstration.ArtifactURL:htmlEncode}</a></tpl></td>',
+                            '</tr>',
+                            '<tr class="skill-grid-demo-detail-row" data-demonstration="{ID}">',
+                                '<td class="skill-grid-demo-detail-data" colspan="6">',
+                                    '<div class="skill-grid-demo-detail-ct">',
+                                        '<tpl if="Demonstration.ArtifactURL">',
+                                            '<div class="skill-grid-demo-artifact">',
+                                                '<strong>Artifact: </strong>',
+                                                '<a href="{Demonstration.ArtifactURL:htmlEncode}" target="_blank">{Demonstration.ArtifactURL:htmlEncode}</a>',
+                                            '</div>',
+                                        '</tpl>',
+                                        '<tpl if="Demonstration.Comments">',
+                                            '<div class="skill-grid-demo-comments">',
+                                                '<strong>Comments: </strong>',
+                                                '{[this.escapeAndNewline(values.Demonstration.Comments)]}',
+                                            '</div>',
+                                        '</tpl>',
+                                        '<div class="skill-grid-demo-meta">',
+                                            'Demonstration #{Demonstration.ID} &middot;&nbsp;',
+                                            '<tpl for="Demonstration.Creator"><a href="/people/{Username}">{FirstName} {LastName}</a></tpl> &middot;&nbsp;',
+                                            '{[fm.date(new Date(values.Demonstration.Created * 1000), "F j, Y, g:i a")]} &middot;&nbsp;',
```

Comment:
I think that time stamps should be converted from seconds to milliseconds before we get to formatting. It's one of those PHP -> JavaScript quirks that are easy to shoot yourself in the foot if you pass through a timestamp to a date object.